package.json

```diff
@@ -16,75 +16,77 @@
     "e2e:ci": "start-server-and-test serve 8080 cy:run"
   },
   "dependencies": {
```

Comment:
you've updated all the major versions of dependencies. It's not the safe change. I'd suggest to rollback.